src/js/components/StatusBar.js

```diff
@@ -1,64 +1,66 @@
 import React from 'react';
-
 import classNames from 'classnames';
 
 class StatusBar extends React.Component {
-  constructor() {
-    super(...arguments);
-  }
-
   static get defaultProps() {
     return {
-      className: 'progress-bar',
-      height: 10
+      className: 'progress-bar flex-box upgrade-progress-bar status-bar',
+      max: 0
```

Comment:
I am not sure about the max. First it actually is min so the total should be at least this number but if the total is higher use the total. I am open to suggestions.